fw/t/unit/http_msg.c

```diff
@@ -1 +0,0 @@
-../../http_msg.c
```

Comment:
Please revert the movement of source files to the unit tests directory

Reply:
Hmmm, file seems to exist in unit tests directory